### PR TITLE
Update rx

### LINKED ISSUE #159: Update RxJava 2 dependency
Problem description
[RxJava 2](https://github.com/ReactiveX/RxJava) dependency should be updated.

### PR DESCRIPTION
**Pull request by Chromicle**
closes #159 
updated rx java dependency

### CODE REVIEW

File: gradle/wrapper/gradle-wrapper.properties [hunk 1]
```diff
@@ -1,6 +1,6 @@
-#Tue Jun 12 21:47:54 IST 2018
+#Sat Mar 02 21:16:46 IST 2019
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-4.4-bin.zip
```
Comment (shobhitagarwal1612) [inline gradle/wrapper/gradle-wrapper.properties]: Please revert this change

**Comment by Chromicle** [comment]
@shobhitagarwal1612 It is directly suggested by the android studio when I sync the project
if not required i will revert

**Comment by shobhitagarwal1612** [comment]
@Chromicle You can still make changes to the same PR by pushing the changes to the same branch. You need not create a new pull request for every new change. If you have any doubts then please feel free to ask questions regarding git or anything else

**Comment by Chromicle** [comment]
I missed the point of making remote branch up to date with main branch so when I push all before changes are get so I pushed but I am unable to made the changes due to to I already commited the changed file
